pkg/yurthub/storage/disk/storage.go

```diff
@@ -30,8 +30,7 @@ import (
 )
 
 const (
-	cacheBaseDir = "/etc/kubernetes/cache/"
```

Comment:
I think we need keep the `cacheBaseDir` as default path for disk storage. if input parameter `dir` is empty, so we can use the default path.

Reply:
`DiskCachePath` has been defined the default path `/etc/kubernetes/cache` to backward compatibility in the `cmd/yurthub/app/options/options.go`.

Reply:
> I think we need keep the `cacheBaseDir` as default path for disk storage. if input parameter `dir` is empty, so we can use the default path.

`DiskCachePath` has been defined default path `/etc/kubernetes/cache/` to `backward compatibility` in the `cmd/yurthub/app/options/options.go`.